### ISSUE TITLE
tg-test.py doesn't match documentation

### ISSUE THREAD

**Issue by DevPGSV** [vysheng/tg#571]
Functions "on_user_update" and "on_chat_update" should have 2 parameters according to the documentation
https://github.com/vysheng/tg/blob/master/README-PY

But they are written without parameters in  tg-test.py: https://github.com/vysheng/tg/blob/master/tg-test.py

Adding those 2 parameters solves the error.

**Comment by Surye** [comment]
Yes, tg-test.py is a mess and changes all the time as I'm testing new stuff, and I am used to ignoring the errors on those. I will work on a cleaned up example python script.
